roqoqo-iqm/src/interface.rs

```diff
@@ -62,51 +64,51 @@ pub struct IqmInstruction {
 // HashMap that associates to each register name the indices in the register that are being affected
 // by measurements. These indices are saved in the order in which the measurement operations appear
 // in the circuit, since this is the order in which the backend returns the results.
-pub(crate) type RegisterMapping = HashMap<String, Vec<usize>>;
```

Comment:
The type of this has been updated, but the corresponding docstring hasn't been. Could you elaborate on this change?

Reply:
I have updated the comment. Previously this type only held information about the measured qubits for each register (which is needed when we don't measure all the qubits, to know which entries of the initialized registers need to be updated with the results). Now it also holds the length of the register, because I moved from initializing the output registers when submitting the circuits, to doing it when we post-process the results, to allow for asynchronous execution.